hawc/apps/common/views.py

```diff
@@ -101,6 +103,36 @@ def create_object_log(verb: str, obj, assessment_id: int, user_id: int):
     reversion.set_comment(comment)
 
 
+def bulk_create_object_log(verb: str, obj_list: Iterable[Any], user_id: int):
+    """
+    Create an object log for each item modified in list.
+
+    Calling this method should be wrapped in a transaction. Does not associate a reversion; bulk
```

Comment:
added a docstring and removed the reversion check; it didn't look like it was actually used